presto-atop/src/main/java/io/prestosql/plugin/atop/AtopSplitManager.java

```diff
@@ -63,7 +65,14 @@ public ConnectorSplitSource getSplits(ConnectorTransactionHandle transactionHand
             ZonedDateTime start = end.minusDays(maxHistoryDays - 1).withHour(0).withMinute(0).withSecond(0).withNano(0);
             while (start.isBefore(end)) {
                 ZonedDateTime splitEnd = start.withHour(23).withMinute(59).withSecond(59).withNano(0);
-                Domain splitDomain = Domain.create(ValueSet.ofRanges(Range.range(TIMESTAMP_WITH_TIME_ZONE, 1000 * start.toEpochSecond(), true, 1000 * splitEnd.toEpochSecond(), true)), false);
+                Domain splitDomain = Domain.create(
+                        ValueSet.ofRanges(Range.range(
+                                TIMESTAMP_TZ_MILLIS,
+                                packDateTimeWithZone(1000 * start.toEpochSecond(), UTC_KEY),
```

Comment:
1000 -> MILLISECONDS_PER_SECOND

Reply:
changed to `.toInstant().toEpochMilli()` (relevant for end time, in case `splitEnd` is not whole seconds (not the case today))